etna/people/models.py

```diff
@@ -57,24 +64,41 @@ class PersonPage(BasePage):
     to link pages to a person, or as a reference to the person.
     """
 
-    role = models.CharField(blank=True, null=True, max_length=100)
-    summary = RichTextField(
-        blank=True, null=True, features=settings.RESTRICTED_RICH_TEXT_FEATURES
-    )
     image = models.ForeignKey(
         get_image_model_string(),
         blank=True,
         null=True,
         on_delete=models.SET_NULL,
         related_name="+",
     )
+    role = models.CharField(blank=True, null=True, max_length=100)
+    summary = RichTextField(
```

Comment:
Should we make this required?

Reply:
They may not want an image (I think Lizzy said she'd like a default/silhouette type thing) but I will make role and summary required 🙂